### ISSUE TITLE
Train loss and test loss

### ISSUE THREAD

**Issue by zhaowenZhou** [ChiWeiHsiao/DeepVO_PyTorch#51]
During training the trian loss is very small like around 0.3, but the smallest test loss is aroung 90.
Then I used trained model provided by [alexart13](https://github.com/alexart13) on test this model on training dataset ['01','02', '05', '08', '09']. The test.py still gave me three-digit even four-digit loss.
Anyone know why?

**Comment by Majal-1036** [comment]
can you help me debug my code, when i run it my training loss comes around 28 for some reason , I downloaded the same repo and set the paths and everything but my training loss comes out to around 28, my img_mean is also different for some reason when I use the preprocessing.py to calculate the mean and std my values are around 80 and not between 0 and 1 although I am using the image_3 data set from KITTI like described in the repo. can someone guide in this regard ?
